admin.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+require_once 'php/dbConnection.php';
+require_once 'php/addMeFunctions.php';
+$db = getDbConn();
+$paragraph = $_POST['add'];
+if(isset($_POST['addParagraph'])){
+    addParagraph($db, $paragraph);
```

Comment:
You could save referencing the $paragraph above by adding it to this parameters of this... i.e. addParagraph($db, $_POST['add']);